db/schema.rb

```diff
@@ -123,8 +123,8 @@
   create_table "distributors_shipping_methods", :force => true do |t|
     t.integer  "distributor_id"
     t.integer  "shipping_method_id"
-    t.datetime "created_at",         :null => false
-    t.datetime "updated_at",         :null => false
+    t.datetime "created_at"
+    t.datetime "updated_at"
```

Comment:
We do have PostgreSQL authentication issues in development :(

In any case, your branch doesn't have any migration. Why are you running `rake db:migrate` at all?